go.mod

```diff
@@ -180,16 +180,16 @@ replace (
 	github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.12.1 // keep this value in sync with sigs.k8s.io/controller-runtime
```

Comment:
This need to be in sync with https://github.com/kubernetes-sigs/controller-runtime/blob/d242fe21e646f034995c4c93e9bba388a0fdaab9/go.mod#L13

and same for `github.com/emicklei/go-restful`